crates/algokit_utils/src/transactions/sender.rs

```diff
@@ -364,10 +365,33 @@ impl TransactionSender {
     /// Send asset opt-out transaction.
     pub async fn asset_opt_out(
         &self,
-        params: AssetOptOutParams,
+        mut params: AssetOptOutParams,
```

Comment:
It's probably better to not have this mutable and clone/change only when close_remainder_to is none.